src/test/tor/minimal/shadow.data.template/hosts/relay4/fingerprint

```diff
@@ -0,0 +1 @@
+relay4middle 7073525D6A7B97E4708CE8B712BAF21049A72168
```

Comment:
I think this should this just be `relay4` since the relay's name is `relay4`?

Reply:
done